question.html

```diff
@@ -56,10 +63,17 @@ <h1 size="10">Questions for {{roomId}}?</h1>&nbsp;
 	<a ng-class="{selected: location.path() == '/users'}" class="btn btn-info" href="#/users">Users</a>
 	<a ng-class="{selected: location.path() == '/about'}" class="btn btn-warning" href="#/about">About</a>
 -->
-
-<small>Share room url: {{absurl}}</small> &nbsp;
-<a class="btn btn-default" href="/index.html">Main</a>
-<button ng-hide="$authData" class="btn btn-primary left" type="button" ng-click="FBLogin()">Want Admin? Sign in with Facebook</button>
+	<div class="container">
```

Comment:
This jumbotron is currently covering almost the whole page, see the full-screen image:

![bildschirmfoto 2015-10-31 um 22 16 16](https://cloud.githubusercontent.com/assets/1174752/10864133/e7ec59e2-801e-11e5-9172-514a4197ed7c.png)

The goal should be to have the questions as close as possible to the top of the page.
Any description of the functionality of resPOSTa shall go on the landing page.
We can decide what to display on the question.html page. 
For now, I suggest removing the whole jumbotron block.